src/player.js

```diff
@@ -35,10 +54,39 @@ export default class Player {
     let audio = document.querySelector("audio");
     input.value = audio.volume;
 
-    input.addEventListener("input", this.boundVolume);
+    return input;
+  }
 
-    let inlineplayer = document.querySelector("div.inline_player");
-    if (!inlineplayer.classList.contains("hidden")) inlineplayer.append(input);
+  static transferPlayButton() {
+    let play_cell = document.querySelector("td.play_cell");
+    play_cell.parentNode.removeChild(play_cell);
```

Comment:
DOM element creation and deletion is a relatively expensive operation. If this is to happen frequently it might be easier to let all elements exist in DOM and simply toggle their visibility instead of recreating them.

Reply:
@emcniece yeah, I don't think this is going to happen much more.

When you say expensive, do you mean computationally or time?

The only thing I worry about with making them invisible is that these elements are actually what the player listens to so my new buttons would have to have their clicks bound to the now invisible elements which I think becomes a bit of work for (potentially) not much benefit.